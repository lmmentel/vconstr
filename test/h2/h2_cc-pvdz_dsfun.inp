&input
	alpha=1.373
	beta=2.0
	df=1.234d0
	dvdmp=0.4d0
	gamma=2.0
	gbasisfile='h2_cc-pvdz.basinfo'
	gdictnfile='h2_cc-pvdz.F10'
	gintegfile='h2_cc-pvdz.F09'
	iprint=2
	itrx=1
	lintsm=.false.
	lrfun=.false.
	lsym=.false.
	nmos=1
	nppr=3
	scfdmp=0.5d0
	thresh=1.0d-6
	title='h2_cc-pvdz'
	tstthr=1.0d-10
/
&occupations
	occmo(1) = 2.0
/
