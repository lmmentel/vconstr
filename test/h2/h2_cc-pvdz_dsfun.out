  Selfconsistent Kohn-Sham calculation 

WARNING; symmetry determination not properly specified


***** title : default 

  norb =  10
  nmos =  1
  orbitals   :  1   
  occupation :  2.00

 Initial potential Vxc =  1.373 X-alpha +  2.000 Vosko-Wilk-Nusair +  2.000 Becke 

 Convergence
  itrx =    1
  threshold = 0.10E-09
  shift   =  1.12
  bounds  =  0.95, 1.05
  density damping :  0.50
  Storage of KS orbitals  :  3

 Number of gridpoints in numerical integration : 8800
 Number of dummy points : 1000

 Electric field : 1.1000x + 2.2000y + 3.3000z
 in -reshapeOrbitals-, nb =           10
 shape of Mpaomo =           10          10
 shape of Maomo =           10          10

Sum of MO occupation numbers =  2.000
Sum of NO occupation numbers =  2.000


                       ================
                       NO's in MO basis
                       ================

              1         2         3         4         5
   1     -0.99997  -0.00000  -0.00455   0.00000   0.00000
   2     -0.00000  -0.71411   0.00000   0.00000   0.00000
   3      0.00640  -0.00000  -0.94374   0.00000   0.00000
   4      0.00000  -0.68977   0.00000   0.00000   0.00000
   5      0.00000   0.00000   0.00000   0.00000   1.00000
   6      0.00000   0.00000   0.00000   1.00000   0.00000
   7      0.00449  -0.00000   0.33065   0.00000   0.00000
   8      0.00000   0.00000   0.00000   0.00000   0.00000
   9      0.00000   0.00000   0.00000   0.00000   0.00000
  10      0.00000  -0.11944   0.00000   0.00000   0.00000


              6         7         8         9        10
   1     -0.00635  -0.00000   0.00000   0.00000   0.00000
   2     -0.00000   0.58865   0.00000   0.00000  -0.37885
   3     -0.33062  -0.00000   0.00000   0.00000  -0.00000
   4      0.00000  -0.49935   0.00000   0.00000   0.52429
   5      0.00000   0.00000   0.00000   0.00000   0.00000
   6      0.00000   0.00000   0.00000   0.00000   0.00000
   7     -0.94374  -0.00000   0.00000   0.00000  -0.00000
   8      0.00000   0.00000  -1.00000   0.00000   0.00000
   9      0.00000   0.00000   0.00000  -1.00000   0.00000
  10      0.00000  -0.63572   0.00000   0.00000  -0.76262


                     ======================
                     CI-density in MO basis
                     ======================

              1         2         3         4         5
   1      1.96486   0.00000   0.00000   0.00000   0.00000
   2      0.00000   0.02216   0.00000   0.00000   0.00000
   3      0.00000   0.00000   0.00612   0.00000   0.00000
   4      0.00000   0.00000   0.00000   0.00308   0.00000
   5      0.00000   0.00000   0.00000   0.00000   0.00308
   6      0.00000   0.00000   0.00000   0.00000   0.00000
   7      0.00000   0.00000   0.00000   0.00000   0.00000
   8      0.00000   0.00000   0.00000   0.00000   0.00000
   9      0.00000   0.00000   0.00000   0.00000   0.00000
  10      0.00000   0.00000   0.00000   0.00000   0.00000


              6         7         8         9        10
   1      0.00000   0.00000   0.00000   0.00000   0.00000
   2      0.00000   0.00000   0.00000   0.00000   0.00000
   3      0.00000   0.00000   0.00000   0.00000   0.00000
   4      0.00000   0.00000   0.00000   0.00000   0.00000
   5      0.00000   0.00000   0.00000   0.00000   0.00000
   6      0.00021   0.00000   0.00000   0.00000   0.00000
   7      0.00000   0.00016   0.00000   0.00000   0.00000
   8      0.00000   0.00000   0.00016   0.00000   0.00000
   9      0.00000   0.00000   0.00000   0.00016   0.00000
  10      0.00000   0.00000   0.00000   0.00000   0.00001


                   =========================
                   HF MO-density in MO basis
                   =========================

              1         2         3         4         5
   1      2.00000   0.00000   0.00000   0.00000   0.00000
   2      0.00000   0.00000   0.00000   0.00000   0.00000
   3      0.00000   0.00000   0.00000   0.00000   0.00000
   4      0.00000   0.00000   0.00000   0.00000   0.00000
   5      0.00000   0.00000   0.00000   0.00000   0.00000
   6      0.00000   0.00000   0.00000   0.00000   0.00000
   7      0.00000   0.00000   0.00000   0.00000   0.00000
   8      0.00000   0.00000   0.00000   0.00000   0.00000
   9      0.00000   0.00000   0.00000   0.00000   0.00000
  10      0.00000   0.00000   0.00000   0.00000   0.00000


              6         7         8         9        10
   1      0.00000   0.00000   0.00000   0.00000   0.00000
   2      0.00000   0.00000   0.00000   0.00000   0.00000
   3      0.00000   0.00000   0.00000   0.00000   0.00000
   4      0.00000   0.00000   0.00000   0.00000   0.00000
   5      0.00000   0.00000   0.00000   0.00000   0.00000
   6      0.00000   0.00000   0.00000   0.00000   0.00000
   7      0.00000   0.00000   0.00000   0.00000   0.00000
   8      0.00000   0.00000   0.00000   0.00000   0.00000
   9      0.00000   0.00000   0.00000   0.00000   0.00000
  10      0.00000   0.00000   0.00000   0.00000   0.00000


                  ===========================
                  Vmo in primitive ao, vmopao
                  ===========================

              1         2         3         4         5
   1     -0.40058  -0.15517   0.70837  -0.79346   0.00000
   2     -0.17996  -1.88096  -0.72727   1.60062   0.00000
   3      0.00000   0.00000   0.00000   0.00000   0.00000
   4      0.00000   0.00000   0.00000   0.00000   0.58395
   5     -0.02215  -0.01309   0.01610   0.38714   0.00000
   6     -0.40058   0.15517   0.70837   0.79346   0.00000
   7     -0.17996   1.88096  -0.72727  -1.60062   0.00000
   8      0.00000   0.00000   0.00000   0.00000   0.00000
   9      0.00000   0.00000   0.00000   0.00000   0.58395
  10      0.02215  -0.01309  -0.01610   0.38714   0.00000


              6         7         8         9        10
   1      0.00000   0.43914   0.00000   0.00000  -2.42562
   2      0.00000  -0.21882   0.00000   0.00000  -0.09741
   3      0.58395   0.00000   0.00000  -0.96791   0.00000
   4      0.00000   0.00000  -0.96791   0.00000   0.00000
   5      0.00000  -0.71084   0.00000   0.00000  -1.95279
   6      0.00000   0.43914   0.00000   0.00000   2.42562
   7      0.00000  -0.21882   0.00000   0.00000   0.09741
   8      0.58395   0.00000   0.00000   0.96791   0.00000
   9      0.00000   0.00000   0.96791   0.00000   0.00000
  10      0.00000   0.71084   0.00000   0.00000  -1.95279

 before -clcvls-, npnt =         9800

     ========================
     Gamess-US job parameters
     ========================

Gamess-US job title                          =  Hydrogen molecule at equlibium in cc-pVDZ basis                                
Number of atoms                              =     2
Total charge                                 =     0
Multiplicity                                 =     1
Number of basis functions                    =    10
Number of orbitals (spherical)               =    10
Total number of electrons                    =     2
Number of alpha electrons                    =     1
Number of beta electrons                     =     1
Number of basis set shells                   =     6
Number of cartesian gaussian basis functions =    10

     ================
     Geometry in a.u.
     ================

     Atom     Charge                X              Y              Z     
     ----   ----------     ---------------------------------------------
     H            1.00          0.00000000     0.00000000    -0.72436800
     H            1.00          0.00000000     0.00000000     0.72436800

     =========
     Basis set
     =========

  Shell   Type   Primitive       Exponent     Contraction Coefficient(s)

H     Atom No.    1     Charge =  1.00

      1     S       1            13.0100000    0.033498726390     1
      1     S       2             1.9620000    0.234800801174     2
      1     S       3             0.4446000    0.813682957883     3
      1     S       4             0.1220000    1.000000000000     4
      1     P       5             0.7270000    1.000000000000     5

H     Atom No.    2     Charge =  1.00

      2     S       1            13.0100000    0.033498726390     6
      2     S       2             1.9620000    0.234800801174     7
      2     S       3             0.4446000    0.813682957883     8
      2     S       4             0.1220000    1.000000000000     9
      2     P       5             0.7270000    1.000000000000    10

     ===================
     Basis set in detail
     ===================

 AO   atom   atom  shell   shell  subshell                                   contraction
indx  name   No.   No.     type     type              exponents              coefficients
-----------------------------------------------------------------------------------
   1  H         1      1     S      S               13.0100000000            0.0334987264
   1  H         1      1     S      S                1.9620000000            0.2348008012
   1  H         1      1     S      S                0.4446000000            0.8136829579
   2  H         1      2     S      S                0.1220000000            1.0000000000
   3  H         1      3     P      X                0.7270000000            1.0000000000
   4  H         1      3     P      Y                0.7270000000            1.0000000000
   5  H         1      3     P      Z                0.7270000000            1.0000000000
   6  H         2      4     S      S               13.0100000000            0.0334987264
   6  H         2      4     S      S                1.9620000000            0.2348008012
   6  H         2      4     S      S                0.4446000000            0.8136829579
   7  H         2      5     S      S                0.1220000000            1.0000000000
   8  H         2      6     P      X                0.7270000000            1.0000000000
   9  H         2      6     P      Y                0.7270000000            1.0000000000
  10  H         2      6     P      Z                0.7270000000            1.0000000000
 before the loop in -clcvls-
 number of points =         9800
 after the loop in -clcvls-

 density :  2.000     integration errors -2.000    -2.000    -2.000     0.000    
WARNING; seems like the basis orbitals are symmetry adapted
 Von Weizacker kinetic energy :  0.00000    
 Orbital Errors upon Integration

  1  1.0     0.50     0.50     0.50      3.0    
  2  1.0     0.50     0.50     0.50      3.0    
  3  1.0     0.50     0.50     0.50      3.0    
  4  1.0     0.50     0.50     0.50      3.0    
  5  1.0     0.50     0.50     0.50      3.0    
  6  1.0     0.50     0.50     0.50      3.0    
  7  1.0     0.50     0.50     0.50      3.0    
  8  1.0     0.50     0.50     0.50      3.0    
  9  1.0     0.50     0.50     0.50      3.0    
 10  1.0     0.50     0.50     0.50      3.0    
 after -clcvls-


 ----- itr    0 -----

      1    1    1    1      0.64978101549981
      2    1    2    1      0.05865517689216
      2    2    1    1      0.38621744271912
      2    2    2    2      0.33610250353465
      3    1    1    1     -0.15638484760109
      3    1    2    2     -0.03563980274218
      3    1    3    1      0.08525057871361
      3    2    2    1      0.02329377288983
      3    2    3    2      0.03797701344214
      3    3    1    1      0.45270089558667
      3    3    2    2      0.32184659271782
      3    3    3    1     -0.07672063263428
      3    3    3    3      0.36123730156690
      4    1    2    1      0.06970668179490
      4    1    3    2     -0.00293499463259
      4    1    4    1      0.11776284129301
      4    2    1    1      0.14004004227622
      4    2    2    2      0.05233952757970
      4    2    3    1     -0.05477043771165
      4    2    3    3      0.06971192391824
      4    2    4    2      0.06214499131034
      4    3    2    1     -0.05077041991679
      4    3    3    2     -0.01245969783532
      4    3    4    1     -0.07398235125683
      4    3    4    3      0.05966697939313
      4    4    1    1      0.57319064753870
      4    4    2    2      0.38155192854083
      4    4    3    1     -0.13103374314341
      4    4    3    3      0.41277578254310
      4    4    4    2      0.13553021016249
      4    4    4    4      0.56750095008565
      5    1    5    1      0.12074265337803
      5    2    5    2      0.01710578798466
      5    3    5    1     -0.02561600766914
      5    3    5    3      0.01449355247635
      5    4    5    2      0.02148335421502
      5    4    5    4      0.03273317172391
      5    5    1    1      0.64268064927460
      5    5    2    2      0.38461542852179
      5    5    3    1     -0.13878384238735
      5    5    3    3      0.44192427651585
      5    5    4    2      0.13380515033205
      5    5    4    4      0.55091676302351
      5    5    5    5      0.69572403730840
      6    1    6    1      0.12074265337803
      6    2    6    2      0.01710578798466
      6    3    6    1     -0.02561600766914
      6    3    6    3      0.01449355247635
      6    4    6    2      0.02148335421502
      6    4    6    4      0.03273317172391
      6    5    6    5      0.03597643627510
      6    6    1    1      0.64268064927460
      6    6    2    2      0.38461542852179
      6    6    3    1     -0.13878384238735
      6    6    3    3      0.44192427651585
      6    6    4    2      0.13380515033205
      6    6    4    4      0.55091676302351
      6    6    5    5      0.62377116475819
      6    6    6    6      0.69572403730840
      7    1    1    1      0.05210519771492
      7    1    2    2     -0.01882419666713
      7    1    3    1     -0.03382604139543
      7    1    3    3      0.02679129397086
      7    1    4    2     -0.00697074939718
      7    1    4    4     -0.01391049794937
      7    1    5    5      0.05605751385166
      7    1    6    6      0.05605751385166
      7    1    7    1      0.07827319912600
      7    2    2    1     -0.04749481374288
      7    2    3    2     -0.01568300569877
      7    2    4    1     -0.05760890216110
      7    2    4    3      0.03432000320269
      7    2    7    2      0.05200939632815
      7    3    1    1     -0.06222384905768
      7    3    2    2     -0.01596319326672
      7    3    3    1      0.03078424181994
      7    3    3    3     -0.03155237499412
      7    3    4    2     -0.01157279321641
      7    3    4    4     -0.03056469505628
      7    3    5    5     -0.05927158509249
      7    3    6    6     -0.05927158509249
      7    3    7    1     -0.03293358106073
      7    3    7    3      0.02559960752318
      7    4    2    1     -0.06786053585484
      7    4    3    2     -0.01774645477068
      7    4    4    1     -0.09005739969041
      7    4    4    3      0.05645717759617
      7    4    7    2      0.07229070655161
      7    4    7    4      0.11098494792177
      7    5    5    1      0.03025453393258
      7    5    5    3     -0.01200204365243
      7    5    7    5      0.03674588660181
      7    6    6    1      0.03025453393258
      7    6    6    3     -0.01200204365243
      7    6    7    6      0.03674588660181
      7    7    1    1      0.62137892381680
      7    7    2    2      0.40608733294940
      7    7    3    1     -0.13815924636909
      7    7    3    3      0.43513197762064
      7    7    4    2      0.14758106580809
      7    7    4    4      0.58492826045964
      7    7    5    5      0.60791493456150
      7    7    6    6      0.60791493456150
      7    7    7    1      0.00409835623540
      7    7    7    3     -0.05226455621767
      7    7    7    7      0.65830869050742
      8    1    5    2     -0.03017533392215
      8    1    5    4     -0.04133280089081
      8    1    8    1      0.05589672111050
      8    2    5    1     -0.05744668980224
      8    2    5    3      0.00649910017520
      8    2    7    5      0.00251339433301
      8    2    8    2      0.03892341669164
      8    3    5    2      0.00247160413443
      8    3    5    4      0.00954383502289
      8    3    8    1     -0.00836128941858
      8    3    8    3      0.00774112900208
      8    4    5    1     -0.07178463044980
      8    4    5    3      0.01624867076103
      8    4    7    5      0.00137879753286
      8    4    8    2      0.04343353274077
      8    4    8    4      0.05859154822001
      8    5    2    1     -0.08068156444370
      8    5    3    2     -0.01776071985409
      8    5    4    1     -0.10813208684128
      8    5    4    3      0.06771843689789
      8    5    7    2      0.06111713002702
      8    5    7    4      0.08852099791530
      8    5    8    5      0.15280867732162
      8    6    8    6      0.01979689844169
      8    7    5    2      0.01679332850972
      8    7    5    4      0.02061324577619
      8    7    8    1     -0.02678156404857
      8    7    8    3      0.00076499768356
      8    7    8    7      0.02963579873627
      8    8    1    1      0.59382929873028
      8    8    2    2      0.39295866893698
      8    8    3    1     -0.11205944083390
      8    8    3    3      0.41950357915849
      8    8    4    2      0.13272782361408
      8    8    4    4      0.54670531241838
      8    8    5    5      0.63291003167511
      8    8    6    6      0.57629337439562
      8    8    7    1      0.00312841554731
      8    8    7    3     -0.03838736576854
      8    8    7    7      0.58662827373292
      8    8    8    8      0.62536794358618
      9    1    6    2     -0.03017533392215
      9    1    6    4     -0.04133280089081
      9    1    9    1      0.05589672111050
      9    2    6    1     -0.05744668980224
      9    2    6    3      0.00649910017520
      9    2    7    6      0.00251339433301
      9    2    9    2      0.03892341669164
      9    3    6    2      0.00247160413443
      9    3    6    4      0.00954383502289
      9    3    9    1     -0.00836128941858
      9    3    9    3      0.00774112900208
      9    4    6    1     -0.07178463044980
      9    4    6    3      0.01624867076103
      9    4    7    6      0.00137879753286
      9    4    9    2      0.04343353274077
      9    4    9    4      0.05859154822001
      9    5    8    6      0.01979689844169
      9    5    9    5      0.01979689844169
      9    6    2    1     -0.08068156444370
      9    6    3    2     -0.01776071985409
      9    6    4    1     -0.10813208684128
      9    6    4    3      0.06771843689789
      9    6    7    2      0.06111713002702
      9    6    7    4      0.08852099791530
      9    6    8    5      0.11321488043824
      9    6    9    6      0.15280867732162
      9    7    6    2      0.01679332850972
      9    7    6    4      0.02061324577619
      9    7    9    1     -0.02678156404857
      9    7    9    3      0.00076499768356
      9    7    9    7      0.02963579873627
      9    8    6    5      0.02830832863974
      9    8    9    8      0.03087641820825
      9    9    1    1      0.59382929873028
      9    9    2    2      0.39295866893698
      9    9    3    1     -0.11205944083390
      9    9    3    3      0.41950357915849
      9    9    4    2      0.13272782361408
      9    9    4    4      0.54670531241838
      9    9    5    5      0.57629337439562
      9    9    6    6      0.63291003167511
      9    9    7    1      0.00312841554731
      9    9    7    3     -0.03838736576854
      9    9    7    7      0.58662827373292
      9    9    8    8      0.56361510716968
      9    9    9    9      0.62536794358618
     10    1    2    1      0.01535377712139
     10    1    3    2     -0.01369085118797
     10    1    4    1      0.04476546094707
     10    1    4    3     -0.03586007688640
     10    1    7    2      0.00776250701990
     10    1    7    4      0.00632918318583
     10    1    8    5     -0.03365164931080
     10    1    9    6     -0.03365164931080
     10    1   10    1      0.06198224975926
     10    2    1    1      0.01785557407864
     10    2    2    2     -0.01732492132505
     10    2    3    1     -0.01699724213675
     10    2    3    3      0.01165441832050
     10    2    4    2     -0.00458600814923
     10    2    4    4     -0.00528742250005
     10    2    5    5      0.01683308949312
     10    2    6    6      0.01683308949312
     10    2    7    1      0.03341525687956
     10    2    7    3     -0.00694717436839
     10    2    7    7     -0.02232556070395
     10    2    8    8     -0.00306139469644
     10    2    9    9     -0.00306139469644
     10    2   10    2      0.02792186025075
     10    3    2    1     -0.02531167533264
     10    3    3    2      0.00976617079325
     10    3    4    1     -0.05505952641125
     10    3    4    3      0.03786833129004
     10    3    7    2      0.01505545120498
     10    3    7    4      0.02212282168248
     10    3    8    5      0.03903184138173
     10    3    9    6      0.03903184138173
     10    3   10    1     -0.04177668434039
     10    3   10    3      0.04155001103158
     10    4    1    1      0.10876205968298
     10    4    2    2      0.00288011784137
     10    4    3    1     -0.06776046936989
     10    4    3    3      0.05977260416018
     10    4    4    2      0.02507108034312
     10    4    4    4      0.06860214030107
     10    4    5    5      0.09442428078632
     10    4    6    6      0.09442428078632
     10    4    7    1      0.05617277400416
     10    4    7    3     -0.02540274211171
     10    4    7    7      0.04578411660080
     10    4    8    8      0.05838482602489
     10    4    9    9      0.05838482602489
     10    4   10    2      0.04346118567112
     10    4   10    4      0.09794515003588
     10    5    5    2      0.00562662707883
     10    5    5    4      0.01156064756515
     10    5    8    1     -0.01495840232864
     10    5    8    3      0.00695010759226
     10    5    8    7     -0.00728196294976
     10    5   10    5      0.01776744600702
     10    6    6    2      0.00562662707883
     10    6    6    4      0.01156064756515
     10    6    9    1     -0.01495840232864
     10    6    9    3      0.00695010759226
     10    6    9    7     -0.00728196294976
     10    6   10    6      0.01776744600702
     10    7    2    1      0.07204532007488
     10    7    3    2      0.02433851870112
     10    7    4    1      0.08804804696303
     10    7    4    3     -0.05446487360169
     10    7    7    2     -0.07444032939982
     10    7    7    4     -0.10162248001782
     10    7    8    5     -0.09346044908531
     10    7    9    6     -0.09346044908531
     10    7   10    1     -0.00343242486116
     10    7   10    3     -0.02857805770093
     10    7   10    7      0.12549784099479
     10    8    5    1     -0.02961394520098
     10    8    5    3      0.01055771161883
     10    8    7    5     -0.02110439902292
     10    8    8    2      0.00496964744238
     10    8    8    4      0.00993645990605
     10    8   10    8      0.02327105928605
     10    9    6    1     -0.02961394520098
     10    9    6    3      0.01055771161883
     10    9    7    6     -0.02110439902292
     10    9    9    2      0.00496964744238
     10    9    9    4      0.00993645990605
     10    9   10    9      0.02327105928605
     10   10    1    1      0.65586733159618
     10   10    2    2      0.41363135642325
     10   10    3    1     -0.16572064447649
     10   10    3    3      0.46037842073558
     10   10    4    2      0.16167598908037
     10   10    4    4      0.62673466643118
     10   10    5    5      0.62219142717170
     10   10    6    6      0.62219142717170
     10   10    7    1      0.00534286075499
     10   10    7    3     -0.05555501951512
     10   10    7    7      0.67277654208968
     10   10    8    8      0.60263314450416
     10   10    9    9      0.60263314450416
     10   10   10    2     -0.00985892663745
     10   10   10    4      0.08990426408477
     10   10   10   10      0.74453083267083

                    ========================
                    Hartree potential matrix
                    ========================

              1         2         3         4         5
   1      1.29956   0.00000  -0.15638   0.00000   0.00000
   2      0.00000   0.77243   0.00000   0.14004   0.00000
   3     -0.15638   0.00000   0.90540   0.00000   0.00000
   4      0.00000   0.14004   0.00000   1.14638   0.00000
   5      0.00000   0.00000   0.00000   0.00000   1.28536
STOP just after -clcvhr-
   6      0.00000   0.00000   0.00000   0.00000   0.00000
   7      0.05211   0.00000  -0.06222   0.00000   0.00000
   8      0.00000   0.00000   0.00000   0.00000   0.00000
   9      0.00000   0.00000   0.00000   0.00000   0.00000
  10      0.00000   0.01786   0.00000   0.10876   0.00000


              6         7         8         9        10
   1      0.00000   0.05211   0.00000   0.00000   0.00000
   2      0.00000   0.00000   0.00000   0.00000   0.01786
   3      0.00000  -0.06222   0.00000   0.00000   0.00000
   4      0.00000   0.00000   0.00000   0.00000   0.10876
   5      0.00000   0.00000   0.00000   0.00000   0.00000
   6      1.28536   0.00000   0.00000   0.00000   0.00000
   7      0.00000   1.24276   0.00000   0.00000   0.00000
   8      0.00000   0.00000   1.18766   0.00000   0.00000
   9      0.00000   0.00000   0.00000   1.18766   0.00000
  10      0.00000   0.00000   0.00000   0.00000   1.31173

